examples/FCCee/fullSim/caloNtupleizer/analysis.py

```diff
@@ -6,19 +6,29 @@
 ROOT.gSystem.Load("libFCCAnalyses")
 ROOT.gErrorIgnoreLevel = ROOT.kFatal
 
+def str2bool(v):
+    if isinstance(v, bool):
+        return v
+    if v.lower() in ('yes', 'true', 't', 'y', '1'):
+        return True
+    elif v.lower() in ('no', 'false', 'f', 'n', '0'):
+        return False
+    else:
+        raise argparse.ArgumentTypeError('Boolean value expected.')
+
 _fcc  = ROOT.dummyLoader
 
 parser = argparse.ArgumentParser()
 
 parser.add_argument("-inputFiles", default = '/afs/cern.ch/user/b/brfranco/work/public/Fellow/FCCSW/key4hep_trial3/FCCSW/Examples/options/output_fullCalo_SimAndDigi_withCluster_MagneticField_False_pMin_10000_MeV_ThetaMinMax_45_135_pdgId_11_pythiaFalse.root', help = "Input rootfiles (can be a single file or a regex)", type = str)
 parser.add_argument("-outputFolder", default = os.path.join("outputs", date.today().strftime("%y%m%d")), help = "Output folder for the rootfiles", type = str)
-parser.add_argument("-storeCellBranches", default = True, help="Whether or not to store cell information", type = bool)
+parser.add_argument("-storeCellBranches", default = True, help="Whether or not to store cell information", type = str2bool)
```

Comment:
Why do you need to convert to bool?
you can use something like this
```parser.add_argument("--storeCellBranches", action='store_true', help="Whether or not to store cell information")```

by default ```args.storeCellBranches``` will be ```False``` and it will turn true only when ```--storeCellBranches``` is in the command line arguments

Reply:
I went for this solution because I found it easier for fine tuning the default scenarios (True/False in the default argpasre will easily lead to a mixture of collection stored and not stored as default behavior). With the store_true option, the default would be to not store the collection and if you want to change the default behavior, you have also to change the if logic inside the code.

Reply:
ok, good. will merge